### ISSUE TITLE
Error: Couldn't find any data in embed page that we know how to parse

### ISSUE THREAD

**Issue by jack-glenn** [microlinkhq/spotify-url-info#109]
**Describe the bug**
I'm using the latest version of this v3.1.9 and I can no longer get previews of Spotify tracks.

**Spotify URL(s)**
https://open.spotify.com/embed/track/0cAiOfc6uxr6NCyQ80ZigK

**Your Code**
N/A

**Error Message**
Error: Couldn't find any data in embed page that we know how to parse

**Comment by Kikobeats** [comment]
Hello,

I double-checked and it's working fine:

<img width="966" alt="CleanShot 2022-12-04 at 10 54 23@2x" src="https://user-images.githubusercontent.com/2096101/205484478-ecbf74ae-cc3a-4c39-a721-f37076c49fb0.png">

https://runkit.com/kikobeats/6376431cd17cd70008f50515

Maybe it was a temporal issue on the Spotify website 🤔

**Comment by jack-glenn** [comment]
Yeah that's strange, I just tried it today and it worked first time without the error that I was getting every time, thanks anyhow.

**Comment by ComputerComa** [comment]
This issue appears to have resurfaced again. If you clone and rerun your runkit, the error shows up again.
[Runkit](https://runkit.com/computercoma/6392035eac544e000939b934)

**Comment by Kikobeats** [comment]
Related https://github.com/microlinkhq/spotify-url-info/actions/runs/3653297374/jobs/6172603196

**Comment by sagivab** [comment]
The same happens in my application (only in production, works great locally).
